frappe/core/doctype/data_import/data_import.py

```diff
@@ -15,7 +15,7 @@
 class DataImport(Document):
 	def autoname(self):
 		if not self.name:
-			self.name = "Import on "+ format_datetime(self.creation)
+			self.name = _("Import on")+" "+format_datetime(self.creation)
```

Comment:
This is the primary key of Data Import. I don't think this should be translated.